test/unit/dashboard_test.rb

```diff
@@ -82,5 +82,14 @@ class DashboardTest < ActiveSupport::TestCase
         assert_equal 0, data.latest_events.length
       end
     end
+
+    test 'latest_events does not fail on ambiguous column name host_id' do
+      data = Dashboard::Data.new
+      #instead of going through the authorizer, force a join that will cause ambiguity.
+      ConfigReport.stubs(:authorized).with(:view_config_reports).returns(ConfigReport.joins(:host => :interfaces))
```

Comment:
Nitpick, please use `expects` instead of `stubs` to prevent the test incorrectly succeeding if the implementation changes for any reason.